libs/components/split-view/src/lib/modules/split-view/split-view-drawer.component.ts

```diff
@@ -82,6 +78,9 @@ export class SkySplitViewDrawerComponent
     return this.#_width || this.widthDefault;
   }
 
+  @ViewChild('drawerRef', { static: true })
+  protected drawerRef!: ElementRef;
```

Comment:
You can use a [signal for this](https://angular.dev/guide/signals/queries):

```
protected drawerRef = viewChild('drawerRef');
```

Reply:
This is neat! I didn't know about it.